### DISCUSSION TITLE
data/2021/06/13/data-event-log-definition/

### DISCUSSION THREAD

**Discussion by zzsza** [zzsza/blog-comments#134] [Comments]
#  데이터 로그 설계, 데이터 로깅, 이벤트 로그 설계, 데이터 QA의 모든 것 

https://zzsza.github.io/data/2021/06/13/data-event-log-definition/

**Comment by zzsza** [comment]
<p>글 읽으면서 로깅할 때 겪었던 시행착오들이 생각나 추억 돋습니다 ㅎㅎㅎ 잘 읽었습니다!</p>

>_This comment was made by Disqus. 2021-06-15 03:11:43 Wootaek Oh_

**Comment by zzsza** [comment]
<p>너무나 좋은 글...고맙습니다.</p>

>_This comment was made by Disqus. 2021-06-16 08:14:01 Seongho Jang_

**Comment by zzsza** [comment]
<p>글 정말 좋네요. 해당 글을 바탕으로 블로깅해서 제가 있는 조직에 공유하고 싶은데 괜찮을까요? 출처도 남기고 추후에 해당 포스트 주소도 남기겠습니다!</p>

>_This comment was made by Disqus. 2021-06-16 09:50:02 배민혁_

**Comment by zzsza** [comment]
<p>저도 이번에 정리하면서 과거 경험들을 다시 떠올렸는데, 추억 돋네요 :) 감사합니다!</p>

>_This comment was made by Disqus. 2021-06-19 11:07:43 Seong Yun  Byeon_

**Comment by zzsza** [comment]
<p>너무 좋은 글이라 해주셔서 감사합니다!</p>

>_This comment was made by Disqus. 2021-06-19 11:07:48 Seong Yun  Byeon_

**Comment by zzsza** [comment]
<p>해당 글을 바탕으로 블로깅이라는 것의 범주에 따라 다를 것 같아요! 제 자료를 2차 가공해서 글 업로드하신다라는 것으로 이해되는데, 거의 유사하게 복기하시는 것보다 제  글 링크를 공유하시는게 더 좋을 것 같아요 :)</p>

>_This comment was made by Disqus. 2021-06-19 11:08:40 Seong Yun  Byeon_

**Comment by zzsza** [comment]
<p>이벤트 설계하는 사람으로서 은근 짜친데 생각보다 어려웠던게 Naming Convention이었어요ㅠㅠㅋㅋ<br>소문자 대문자 구분할 것이냐 말 것이냐, 띄어쓰기를 쓸 것이냐, 언더 바로 통일할 것이냐, 추상화 수준은 어디까지 할 것이냐 등등..그거 할 땐 누가 좀 해줬으면 좋겠다란 생각도 들었습니다 ㅋㅋㅋ</p><p>그리고 데이터 QA 부분이 좀 더 궁금하네요. 그냥 이벤트 설계한게 잘 들어오는지 정도는 늘 체크하는데, 그 정도만으로는 부족한 느낌? PA툴로 이벤트를 볼 때 종종 유실되는 경우들도 있다고 해서.. 그런 건 QA 단계 때 어떻게 미리 예방할 수 있을지 궁금합니다.</p>

>_This comment was made by Disqus. 2021-06-19 05:43:58 Donghyun  Yu_

**Comment by zzsza** [comment]
<p>그리고 좋은 글 너무너무 감사합니다 성윤님~!</p>

>_This comment was made by Disqus. 2021-06-19 05:44:08 Donghyun  Yu_

**Comment by zzsza** [comment]
<p>우왕~ 좋은글 감사합니다.</p>

>_This comment was made by Disqus. 2021-06-21 09:51:30 ByeongKeon Kim_

**Comment by zzsza** [comment]
<p>잘 정리해주신 글 잘 봤습니다~! 감사합니다.</p>

>_This comment was made by Disqus. 2022-02-25 03:30:44 Dennis Lee_

**Comment by umgong** [comment]
안녕하세요. 
저는 웹서비스 QA입니다. 덕분에 영역을 확장해서 데이터 QA도 중요한 영역이라는 생각도 갖게되고 관심이 생겼습니다.
혹시 인프런 강의 내용에서 쉽게 데이터 QA를 진행할 수 있는 방법도 나올까요? 기술적인 이해도가 높지 않은데 데이터 QA의 역할 중 일부 쉬운 부분이라도 할 수 있는 방법이 궁금해서요.
초보 데이터 qa방법도 알려주실 수 있을까요ㅎㅎㅎ
